app/components/common/export.vue

```diff
@@ -54,24 +54,26 @@
-                                                        <th style="white-space:nowrap;" class="tableexport-string">{{$t('info3')}} </th>
-                                                        <th style="white-space:nowrap;" class="tableexport-string">{{$t('info4')}} </th>
-                                                        <th style="white-space:nowrap;" class="tableexport-string">{{$t('publishedDate')}}</th>
+                                                        <th class="tableexport-string">{{t('type')}}</th>
+                                                        <th class="tableexport-string">{{t('uid')}}</th>
+                                                        <th class="tableexport-string">{{t('government')}}</th>
+                                                        <th class="tableexport-string" style="max-width:30%">{{t('title')}} </th>
+                                                        <th style="white-space:nowrap;" class="tableexport-string">{{t('info1')}} </th>
+                                                        <th style="white-space:nowrap;" class="tableexport-string">{{t('info2')}} </th>
+                                                        <th style="white-space:nowrap;" class="tableexport-string">{{t('info3')}} </th>
+                                                        <th style="white-space:nowrap;" class="tableexport-string">{{t('info4')}} </th>
+                                                        <th style="white-space:nowrap;" class="tableexport-string">{{t('publishedDate')}}</th>
                                                     </tr>
                                                 </thead>
                                                 <tbody>
                                                     <tr v-for="(row, index) in downloadDocs" :key="index">
                                                         
                                                         <td class="tableexport-string">{{row.rec_schema}}</td>
                                                         <td class="tableexport-string">
-                                                            <a rel="noopener" target="_blank" :href="`database/${$options.filters.encodeURIComponent($options.filters.capitalize((row.rec_uniqueIdentifier||'')))}`">
-                                                                {{(row.rec_uniqueIdentifier||'')|capitalize}}
+                                                            <!-- <a rel="noopener" target="_blank" :href="`database/${$options.filters.encodeURIComponent($options.filters.capitalize((row.rec_uniqueIdentifier||'')))}`"> -->
+                                                                <!-- ToDo: ?? -->
+                                                            <a v-if="row.rec_uniqueIdentifier" rel="noopener" target="_blank" :href="`database/${(row.rec_uniqueIdentifier).toUpperCase()}`">
```

Comment:
I think it is missing `encodeURIComponent`
```suggestion
                                                            <a v-if="row.rec_uniqueIdentifier" rel="noopener" target="_blank" :href="`database/${encodeURIComponent((row.rec_uniqueIdentifier).toUpperCase())}`">
```

Reply:
added encodeURIComponent